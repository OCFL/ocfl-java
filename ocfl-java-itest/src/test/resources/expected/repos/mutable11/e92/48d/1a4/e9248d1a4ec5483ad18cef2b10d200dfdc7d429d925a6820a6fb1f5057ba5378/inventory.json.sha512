4f64704b3f59181a59a2b27654769bac427a6671a76416776c2dc4e9bd8f5e8f0801af4b11f3286368f8c41b07d26be853d1c2108a56f0c502cb644301f9c656  inventory.json
