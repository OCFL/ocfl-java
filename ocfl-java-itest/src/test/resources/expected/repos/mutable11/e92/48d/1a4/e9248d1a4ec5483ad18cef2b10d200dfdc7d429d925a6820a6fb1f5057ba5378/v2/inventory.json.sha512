e65c772ae553834ae2269bb3f320f17a9dc3cdf0e0768f8958c03c964ac1a175995ded9a8eafd39a475bf7bc070bac853fe1f8c4c594ed19ecd5052727a2af72  inventory.json
