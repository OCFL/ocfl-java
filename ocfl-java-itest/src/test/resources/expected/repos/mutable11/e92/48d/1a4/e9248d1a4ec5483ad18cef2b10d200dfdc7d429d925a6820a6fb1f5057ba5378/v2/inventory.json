{
  "id" : "object_1",
  "type" : "https://ocfl.io/1.1/spec/#inventory",
  "digestAlgorithm" : "sha512",
  "head" : "v2",
  "contentDirectory" : "content",
  "fixity" : { },
  "manifest" : {
    "7b54b66836c1fbdd13d2441d9e1434dc62ca677fb68f5fe66a464baadecdbd00576f8d6b5ac3bcc80844b7d50b1cc6603444bbe7cfcf8fc0aa1ee3c636d9e339" : [ "v2/content/r1/info_1.txt" ]
  },
  "versions" : {
    "v1" : {
      "created" : "2019-08-05T15:57:53Z",
      "message" : "Auto-generated empty object version.",
      "user" : {
        "name" : "ocfl-java",
        "address" : "https://github.com/OCFL/ocfl-java"
      },
      "state" : { }
    },
    "v2" : {
      "created" : "2019-08-05T15:57:53Z",
      "state" : {
        "7b54b66836c1fbdd13d2441d9e1434dc62ca677fb68f5fe66a464baadecdbd00576f8d6b5ac3bcc80844b7d50b1cc6603444bbe7cfcf8fc0aa1ee3c636d9e339" : [ "info_1.txt" ]
      }
    }
  }
}